src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,103 @@
+package duke;
+
+import duke.command.*;
```

Comment:
Will it be better to list out all the classes in the package following the coding standards?